### PR TITLE
DevEx: alert of critical events

### PR DESCRIPTION
**Pull request by sutt0n**
When an error/exception is thrown on any lambda function, an e-mail will be dispatched to the configured SNS topic.

### CODE REVIEW

**Review by kkoskelin — APPROVED** [review]

**Review by wbyoko — APPROVED** [review]
whats the protocol for adding more emails

File: efcms-service/serverless.yml [hunk 1]
```diff
@@ -54,13 +54,14 @@ custom:
...
+      - errorExceptions
     definitions:
       errorExceptions:
-        description: 'Internal Lambda Alarm'
+        description: 'Internal Error Exception Alarm'
         namespace: 'AWS/Lambda'
         metric: Errors
         threshold: 1
-        statistic: Minimum
+        statistic: Sum
```
Comment (codyseibert) [inline efcms-service/serverless.yml:64]: what's the context behind switching these?

**Comment by sutt0n** [comment]
@wbyoko To add more emails under the `topics` object, you can add another entry under `notifications` like so:

```yaml
notifications:
  - protocol: email
    endpoint: jsutton@flexion.us
  - protocol: email
    endpoint: another-email@another-site.com
```

File: efcms-service/serverless.yml (hunk 1 above)
Comment (sutt0n) [reply efcms-service/serverless.yml:64]: After testing it with `Sum` and observing the errors being e-mailed, I decided to go with `Sum` instead. If we changed the `comparisonOperator` below from `GreaterThanThreshold` to `GreaterThanOrEqualToThreshold`, we may see the same result as well.

**Review by codyseibert — APPROVED** [review]

**Pull request merged**